lib/monkey-mailer/adapters/mandrilapi.rb

```diff
@@ -59,6 +56,27 @@ def send_email(email)
         raise MonkeyMailer::DeliverError.new("Mandril response.code not equal to 200") unless response.code.to_i == 200
         puts "Response #{response.code} #{response.message}: #{response.body}"
       end
+
+      def parse_recipients(recipients_list, names_list = "", type = :to)
+        recipients = recipients_list.split(",")
+        names      = names_list.to_s.split(",")
```

Comment:
I added that `to_s` at last minute, because if you don't have a `to_name` method/attribute in your `email` object, it will receive `nil` there. For example:

```
require './lib/monkey-mailer/adapters/mandrilapi'

adapter = MonkeyMailer::Adapters::MandrilAPI.new(:mandril_api_key => "my-mandrill-api-key")

email = OpenStruct.new({
  :to_email => "dest@example.com,dest_2@example.com",
  :from_name => "Monkey Mailer Test Script",
  :from_email => "mm@monkey.com",
  :subject => "Test From Mandrill",
  :body => "<h1>Sample test</h1>",
  :attachments => []
})

puts adapter.send_email(email)
``` 

However, I'm not sure if that could be a real use case, since I'm not using the `MonkeyMailer` here but the adapter directly.

IMO, it doesn't hurt keeping it, and gives the chance of using the adapter directly if you need to for a simple scenario.